docs/src/getting_started/Android/android.md

```diff
@@ -114,6 +123,10 @@ like this:
 
 ```
 
+```admonish warning title="Sharp edge"
+You will need to set the `ndkVersion` to one you have installed, go to "**Tools, SDK Manager, SDK Tooks**" and check "**Show Package Details**" to get your installed version, or to install the version matching `build.gradle` above.
+```
```

Comment:
I'm not sure if there's an easier way around this, like some way to say `"latest"`? The error message if the version mismatches is an unhelpful `NDK is not installed`

Reply:
No this is good. Thanks!